usercp.php

```diff
@@ -1462,14 +1462,14 @@
         );
         $db->update_query("users", $updated_avatar, "uid='".$mybb->user['uid']."'");
         remove_avatars($mybb->user['uid']);
-    } elseif($_FILES['avatarupload']['name']) // upload avatar
+    } else if ($_FILES['avatarupload']['name']) // upload avatar
```

Comment:
Since we've changed the convention here, can we also move the `{` to the same line?